apps/src/templates/teacherDashboard/TeacherDashboardHeader.jsx

```diff
@@ -26,12 +79,54 @@ export default class TeacherDashboardHeader extends React.Component {
           isRtl={true}
           chevronSide="left"
         />
-        <div style={styles.headerContainer}>
-          <h1>{this.props.sectionName}</h1>
-          <SelectSectionDropdown />
+        <div style={styles.header}>
+          <div>
+            <h1 style={styles.shortH1}>{this.selectedSection.name}</h1>
+            <div>
+              <span style={styles.sectionPrompt}>Assigned to: </span>
```

Comment:
we'll want to make this string and "Edit section details" below translatable by putting them in the i18n pipeline if they aren't already there.

Reply:
Will do 👍